pandas/tests/io/test_packers.py

```diff
@@ -934,3 +934,7 @@ def test_msgpack_period_freq(self):
         s = Series(np.random.rand(5), index=date_range('20130101', periods=5))
         r = read_msgpack(s.to_msgpack())
         repr(r)
+
+    def test_msgpack_file_not_found(self):
+        bad_path = '1/2/3/4/path/does/not/exist'
```

Comment:
can you add the issue number here as a comment

Reply:
I removed this test and added the issue number inline where the fix was done.